oauth2/oauth2-client/src/main/java/org/springframework/security/oauth2/client/JdbcOAuth2AuthorizedClientService.java

```diff
@@ -107,6 +112,8 @@
 
 	protected Function<OAuth2AuthorizedClientHolder, List<SqlParameterValue>> authorizedClientParametersMapper;
 
+	protected final LobHandler lobHandler;
```

Comment:
Feel free to add `@author` tag at class level

Reply:
Thank you!